lib/dealog_backoffice/messages/projectors/message_service.ex

```diff
@@ -47,6 +47,7 @@ defmodule DealogBackoffice.Messages.Projectors.MessageService do
           headline: event.title,
           description: event.body,
           ars: "059580004004",
+          organization: "DEalog Team",
```

Comment:
@ctietze This is the relevant change. Note that currently no metadata is saved to the message. Once this is done the hard coded stuff goes away.